pipe/pipe.py

```diff
@@ -84,26 +84,27 @@ def build_report_data(failure_count):
 
         report_id = str(uuid.uuid4())
 
-        bitbucket_api = Bitbucket(proxies={"http": 'http://host.docker.internal:29418'})
-
-
-        failures = read_failures_from_file(
-                f"{self.out_path}dependency-check-junit.xml"
-                )
-
-        bitbucket_api.create_report(
-                "OWASP Dependency Scan",
-                "Results produced when scanning {self.scan_path} for known OWASP vulnerabilities." ,
-                "SECURITY" ,
-                report_id,
-                "owasp-dependency-check-pipe" ,
-                "FAILED" if len(failures) else "PASSED",
-                f"https://bitbucket.org/{self.bitbucket_workspace}/{self.bitbucket_repo_slug}/addon/pipelines/home#!/results/{self.bitbucket_pipeline_uuid}/steps/{self.bitbucket_step_uuid}/test-report",
-                build_report_data(len(failures)),
-                self.bitbucket_workspace,
-                self.bitbucket_repo_slug,
-                self.bitbucket_commit
-                )
+        if self.bitbucket_repo == 'Unknown Project':
```

Comment:
Isn't this doing the opposite of what we want?

Reply:
Oops. Good catch